src/log_prob.jl

```diff
@@ -0,0 +1,438 @@
+# Functions to compute the log probability of star parameters and galaxy
+# parameters given pixel data
+
+using Distributions
+import ..SensitiveFloats: SensitiveFloat, zero_sensitive_float, clear!
+
+include("bivariate_normals.jl")
+include("log_prob_util.jl")
+EPS = 1e-6
```

Comment:
Non-constant globals really slow things down in julia, for some reason. Try `const EPS = 1e-6`.